libs/langchain-anthropic/src/chat_models.ts

```diff
@@ -502,19 +703,142 @@ export class ChatAnthropicMessages<
       });
     }
     const makeCompletionRequest = async () =>
-      this.batchClient.messages.create({
-        ...request,
-        ...this.invocationKwargs,
-      } as AnthropicMessageCreateParams);
+      this.batchClient.messages.create(
+        {
+          ...request,
+          ...this.invocationKwargs,
+        } as AnthropicMessageCreateParams,
+        options
+      );
     return this.caller.callWithOptions(
-      { signal: options.signal },
+      { signal: options.signal ?? undefined },
```

Comment:
Why?

Reply:
signal in caller has to be signal or undefined, their sdk types it as signal, null or undefined and I'm using their types for this.